src/corelibs/U2Gui/src/util/GUIUtils.cpp

```diff
@@ -226,9 +226,10 @@ const QColor GUIUtils::OK_COLOR = QColor(255,255,255);
 
 void GUIUtils::setWidgetWarning(QWidget *widget, bool value) {
     QColor color = value ? WARNING_COLOR : OK_COLOR;
```

Comment:
Why do you use `background: rgb(r,g,b)` format of a color in style definition, but don't use `background: #RRGGBB` format? As I understand the result of the `QColor.name()` method returns a color name in such format: `#RRGGBB`. And you won't have to convert the color components separately as you do in lines below.

Reply:
Documentation links:
- https://doc.qt.io/archives/qt-4.8/qcolor.html#name
- https://doc.qt.io/qt-5/qcolor.html#name